parquet/src/arrow/array_reader/byte_view_array.rs

```diff
@@ -641,8 +643,9 @@ impl ByteViewArrayDecoderDelta {
                 if bytes.len() > 12 {
                     // only copy the data to buffer if the string can not be inlined.
                     array_buffer.extend_from_slice(bytes);
+                } else {
+                    utf8_validation_buffer.extend_from_slice(bytes);
```

Comment:
this only copies the strings once (but then we have to call validate twice on the two different buffers)

Reply:
thanks!